README.md

```diff
@@ -32,26 +39,29 @@ Output:
 dtype: int32
 ```
 
-For additional examples, browse our complete [API documentation](https://docs.rapids.ai/api/cuml/stable/), or check out our more detailed [walkthrough notebooks](https://github.com/rapidsai/notebooks/tree/master/cuml).
+For additional examples, browse our complete [API
+documentation](https://docs.rapids.ai/api/cuml/stable/), or check out our more
+detailed [walkthrough
+notebooks](https://github.com/rapidsai/notebooks/tree/master/cuml).
 
```

Comment:
Should we add a link to the notebooks-extended repository as well for completeness?